job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -61,6 +64,14 @@ object WebApi {
     Map(ResultKey -> result)
   }
 
+  def resultToByteIterator(jobReport: Map[String, Any], result: Iterator[_]): Iterator[_] = {
+    ResultKeyStartBytes.toIterator ++
+      (jobReport.map(t => Seq(AnyJsonFormat.write(t._1).toString(),
```

Comment:
But I'm trying to avoid converting the lot including result into JsObject tokens, since the whole thing including result in the middle of jobReport JsObject needs to be chunked in the response. Unless I'm missing something I can't see how I can do it directly with `AnyJsonFormat.write(jobReport)` without performing conversion of the whole thing in strict way. I agree the things looks a bit ugly, but unless I modify AnyJsonFormat in some way, I'm not sure how to do it as it stands.